homeassistant/components/homekit/__init__.py

```diff
@@ -98,6 +100,9 @@
                 vol.Optional(CONF_SAFE_MODE, default=DEFAULT_SAFE_MODE): cv.boolean,
                 vol.Optional(CONF_FILTER, default={}): FILTER_SCHEMA,
                 vol.Optional(CONF_ENTITY_CONFIG, default={}): validate_entity_config,
+                vol.Optional(
+                    CONF_ZEROCONF_DEFAULT_INTERFACE, default=False
```

Comment:
Create a constant for this default value.

```suggestion
                    CONF_ZEROCONF_DEFAULT_INTERFACE, default=DEFAULT_ZEROCONF_DEFAULT_INTERFACE
```